test/cases/5.7_math.yaml

```diff
@@ -31,11 +103,86 @@ tests:
     expression: Math.n3.log([3, 5])
     error: true
   
+  - desc: '5.7.7  power(exponent : Integer | Decimal) : Integer | Decimal'
+  - desc: '** Can raise input number to the power of given degree'
+    expression: Math.n4.power(2)
+    result: [64]
+  - desc: '** Empty result if the power cannot be represented'
+    expression: Math.n6.power(0.5)
+    result: []
+  - desc: '** Empty result when raising empty collection to a power'
+    expression: Math.n2.power(8)
+    result: []
+  - desc: '** Empty result when raising collection to the empty power'
+    expression: Math.n3.power(n2)
+    result: []
+  - desc: '** Error raising to a power if the input collection contains multiple items'
+    expression: Math.arr.power(8)
+    error: true
+  - desc: '** Error raising to a power due to too many input parameters'
+    expression: Math.n3.power([3, 5])
```

Comment:
Needs a test for negative numbers:
(-1).power(0.5)
Also, please add (-1).power(1.5)

Reply:
Note that it is not necessary to start all expressions with a resource name (Math).  You can just start with (-1)....  An advantage is that you don't have to look at the bottom of the file to see what the expression is operating on.

Reply:
I will directly use the values instead of variables.
But if I start expression with negative number ((-1).power(1.5)), it throws an error "No PolarityExpression evaluator".

Reply:
Right.  See pull request #13, which implements PolarityExpression.  For now, maybe for negative numbers just leave them in the resource.